plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/handle-prototype.js

```diff
@@ -0,0 +1,37 @@
+function handlePrototype(url, crumb) {
+    buildFormTree(document.forms.config);
+    // TODO JSON.stringify fails in some circumstances: https://gist.github.com/jglick/70ec4b15c1f628fdf2e9 due to Array.prototype.toJSON
+    // TODO simplify when Prototype.js is removed
+    const json = Object.toJSON ? Object.toJSON(JSON.parse(document.forms.config.elements.json.value).prototype) : JSON.stringify(JSON.parse(document.forms.config.elements.json.value).prototype);
+    if (!json) {
+        return; // just a separator
+    }
+
+    const xhr = new XMLHttpRequest();
```

Comment:
Is there a reason this is using a different API than the script previously (at least as of #702) inside `index.jelly`, using `fetch`?

Reply:
It wasn't working anymore locally using `fetch` so I moved with that other HTTP library.

Reply:
> It wasn't working anymore locally using `fetch`

But why would moving the code from inline `<script>` to an adjunct cause that? It apparently works in, e.g., https://github.com/jenkinsci/credentials-plugin/pull/443 so what went wrong here?

Reply:
After some investigations, I've moved back to `fetch`!
see https://github.com/jenkinsci/workflow-cps-plugin/pull/599/commits/196f96ca5c3bc698e5a9c8d47e2e6ed4287dc701